numpy/core/tests/test_ufunc.py

```diff
@@ -1758,7 +1778,7 @@ def test_reduce_arguments(self):
         # too little
         assert_raises(TypeError, f)
         # too much
-        assert_raises(TypeError, f, d, 0, None, None, False, 0, 1)
+        assert_raises(TypeError, f, d, 0, None, None, False, 0, True, 1)
```

Comment:
I think these arguments are swapped

Reply:
It was for "too many arguments" which didn't work any more. But if we go with keyword only, I can revert this...

Reply:
Oh, I see - the `1` is the extra. You're right, what you have is correct.